infoblox/resource_infoblox_a_record.go

```diff
@@ -79,7 +79,11 @@ func resourceARecordCreate(d *schema.ResourceData, m interface{}) error {
 	fqdn := d.Get("fqdn").(string)
 	ipAddr := d.Get("ip_addr").(string)
 	if ipAddr == "" && cidr == "" {
-		return fmt.Errorf("error creating A-record: 'ip_addr' is empty and either 'cidr' or 'network_view' values are absent")
+		return fmt.Errorf("error creating A-record: either of 'ip_addr' and 'cidr' values is required")
+	}
+
+	if ipAddr != "" && cidr != "" {
+		return fmt.Errorf("error updating A-record: only one of 'ip_addr' and 'cidr' values is allowed to be defined")
```

Comment:
We have to reformulate updating to creating in the error message as condition comes under creation,  and also in  regexp in the test file, which follows same error message.

Reply:
ok